### PR TITLE
New custom platforms for HTTP

### PR DESCRIPTION
**Pull request by petrsnd**

### CODE REVIEW

File: SampleScripts/WordPressHttp.json [hunk 1]
```diff
@@ -417,7 +417,7 @@
 			},
 			{
 				"Condition": {
-					"If": "UserId.Equals(\'null\')",
```
Comment (petrsnd) [inline SampleScripts/WordPressHttp.json]: @NathanTurnbow Can you confirm I've done the right thing here?  Why are we looking for "null" as a string anyway?

I'm going to start reformatting the code.

**Comment by petrsnd** [comment]
These scripts need to be re-validated.

File: SampleScripts/CustomTwitter.json [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            "Do": [
+              { "Log": { "Text": "Authentication was successful, but the account is locked (maybe temporarily)" } },
+              { "Status": { "Type": "Checking", "Percent": 80, "Message": { "Name": "AccountLocked", "Parameters": [ "%AccountUserName%" ] } } },
+              { "Return": { "Value": false } }
+            ]
+          }
+        }
+      },
+      { "Function": { "Name": "ChangePasswordInternal", "Parameters": ["%AccountUserName%"], "ResultVariable": "ChangeResult" } },
+      { "Return": { "Value": "%ChangeResult%" } }
```
Comment (RobertCroteau) [inline SampleScripts/CustomTwitter.json]: Should the new password be tested like for Facebook?

File: SampleScripts/CustomTwitter.json [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    ]
+  },
+  "Functions": [
+    {
+      "Name": "Init",
+      "Do": [
+        { "SetItem": { "Name": "GLOBAL:LoginChallenge", "Value": false } },
+        { "SetItem": { "Name": "GLOBAL:AccountLocked", "Value": false } },
+        { "SetItem": { "Name": "GLOBAL:ChallengeType", "Value": "" } }
```
Comment (RobertCroteau) [inline SampleScripts/CustomTwitter.json]: Other locations use Global:  
Should we be consistent

**Comment by RobertCroteau** [comment]
As a personal preference, I'm not keen on single lines with multiple nested object defined.

**Comment by RobertCroteau** [comment]
Other than that 👍

**Comment by petrsnd** [comment]
@miroberts is going to file bugs for the use of the global flag and we will try to eliminate that as bug fixes in a future release.

**Pull request merged**